playbooks/cp4s_get_open_cases.yml

```diff
@@ -0,0 +1,11 @@
+# The MIT License
+#  
+# Copyright 2021 Ryan-Gordon1
```

Comment:
Add your name to the copyright for this file. ;)

Reply:
Just a general comment, put your name in the copyright on every file you touch as anything you contribute makes you part of the project

Reply:
Whoops! Sorry, ill update the file. Thanks